app/controllers/contacts_controller.rb

```diff
@@ -1,15 +1,26 @@
 class ContactsController < ApplicationController
   before_action :set_contact, only: %i[edit update destroy]
-  before_action :set_contact_type_options, only: %i[create edit new update]
```

Comment:
You could bring back the `set_contact_type_options` before_action for :new and :new_details